tests/utils.py

```diff
@@ -135,6 +136,30 @@ def running_on_jenkins() -> bool:
     return RUNNING_ON_JENKINS
 
 
+def running_in_multi_threaded_mode() -> bool:
+    """Whether tests are currently running in multi-threaded mode."""
+    return os.getenv("SNOWPARK_MULTITHREADING_MODE") == "true"
+
+
+def multithreaded_run(num_threads: int) -> None:
```

Comment:
nit: you can make default num_threads=5 so that you do not need to specify num_thread in each call which defaults to 5